src/OrchardCore.Modules/OrchardCore.Forms/Models/SelectPart.cs

```diff
@@ -6,6 +6,7 @@ public class SelectPart : ContentPart
     {
         public SelectOption[] Options { get; set; }
         public string DefaultValue { get; set; }
+        public SelectEditorOption Editor { get; set; }
```

Comment:
Set Default value to Dropdown, so the exiting items' wont break with NRE

Reply:
Same here `SelectEditorOption` is an enum with `None` is the default value. so it won't break anything

Reply:
done.